documentation/reference_doc/5_outputs.tex

```diff
@@ -65,12 +65,19 @@ \subsection{On Site Fossil Fuel Emissions}
 Natural gas, propane, and fuel oil emissions use the emission factors in \textit{Table 7.1.2(1) of draft National Average Emission Factors for Household Combustion Fuels} defined in \textit{ANSI/RESNET/ICCC 301-2022 Addendum B-2022 Standard for the Calculation and Labeling of the Energy Performance of Dwelling and Sleeping Units using an Energy Rating Index}. Natural gas emissions include both combustion and pre-combustion emissions (e.g., methane leakage for natural gas).
 
 On-Site Fossil Fuel Emissions Factors:\\
-Natural gas: 147.3 lb/MMBtu (228.0 kg/MWh)\\ 
-Propane: 177.8 lb/MMBtu (275.7 kg/MWh)\\ 
-Fuel oil: 195.9 lb/MMBtu (303.2 kg/MWh)\\ 
+Natural gas: 147.3 lb/MMBtu (228.0 kg/MWh)\\
+Propane: 177.8 lb/MMBtu (275.7 kg/MWh)\\
+Fuel oil: 195.9 lb/MMBtu (303.2 kg/MWh)\\
 
 \subsection{District Energy Emissions}
-ComStock currently does not model emissions from district energy systems, as there is considerable variation by location and type of district system.
```

Comment:
I think this bit about district emissions is duplicative, so merge this PR after the district system emissions is merged.

Reply:
yes it is; there is a note at the top to merge after #302